metricflow/model/validations/measures.py

```diff
@@ -455,14 +455,29 @@ def validate_model(model: UserConfiguredModel) -> List[ValidationIssueType]:  #
                                 ),
                             )
                         )
-                elif (
-                    measure.agg != AggregationType.PERCENTILE
-                    and measure.agg_params
-                    and (
-                        measure.agg_params.percentile
-                        or measure.agg_params.use_discrete_percentile
-                        or measure.agg_params.use_approximate_percentile
-                    )
+                elif measure.agg == AggregationType.MEDIAN:
```

Comment:
With our current setup this will never happen but I think it's good to have this here as well. It might be useful for models pulled in from dbt configs, for example.